connaisseur/notary_api.py

```diff
@@ -97,6 +97,15 @@ def get_trust_data(host: str, image: Image, role: TUFRole, token: str = None):
     return TrustData(data, role.role)
 
 
+def get_delegation_trust_data(
+    host: str, image: Image, role: TUFRole, token: str = None
+):
+    try:
+        return get_trust_data(host, image, role, token)
+    except Exception:
```

Comment:
You probably want only `connaisseur.exceptions.NotFoundException`?

Reply:
welp. the problem is that we are actually getting a `500 internal server error` back, even though the problem should be, that the requested files do not exist, meaning theoretically it should be `404 not found`. so the `NotFoundException` is not raised here. instead of grabbing `500` now in the `get_trust_data` function, we decided to grab all exceptions instead, as we are not sure what other kind of other weird behavior could come up further down the line (since returning 500 in case is already weird).

Reply:
hm. Maybe catch all in production, but only specific stuff in debug mode? That way you can learn more about exceptions in debug mode.

Reply:
done.